apis/control/v1alpha1/cassandratask_types.go

```diff
@@ -0,0 +1,187 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// TODO Labels to add that indicate the status of the task? So they can be searched? Like jobs have.
```

Comment:
Well, this is actually implemented for the active status, not otherwise. The original behavior did not need it, this only because it's no longer tied to the CassandraDatacenter directly.

Reply:
Also, forgot.. this is task label, not pod annotations (those still remain annotations).